src/condor_utils/docker-api.cpp

```diff
@@ -922,7 +922,21 @@ sendDockerAPIRequest( const std::string & request, std::string & response ) {
 	memset(&sa, 0, sizeof(sa));
 
 	sa.sun_family = AF_UNIX;
-	strncpy(sa.sun_path, "/var/run/docker.sock",sizeof(sa.sun_path) - 1);
+	// Default docker socket path location
+	std::string docker_socket_path = "/var/run/docker.sock";
+	// unless overridden by env var
+	std::string docker_host;
+	if (getenv("DOCKER_HOST")) {
+		docker_host = getenv("DOCKER_HOST");
```

Comment:
This makes me sad, but I'm not sure that
```suggestion
	if ((docker_host = getenv("DOCKER_HOST"))) {
```
is any better.

Reply:
Yeah, I went back and forth on that.  I don't love the assignment in a predicate.